addon/components/select-2.js

```diff
@@ -1,4 +1,5 @@
 import Ember from "ember";
+import DS from "ember-data";
```

Comment:
I think you can use it without ember data, but I will test it without ember data too to make sure.